developer/integrations/bigcommerce-catalyst.mdx

```diff
@@ -0,0 +1,137 @@
+  4. Install Catalyst Dependencies.
+  5. Set up environment variables.
+
+```
+npm create @bigcommerce/catalyst@latest
+```
+
+Once you run the CLI, you effectively have a Catalyst Next.js application created. The remaining steps will walk you through integrating your Catalyst application with Makeswift.
+
+  </Tab>
+  <Tab title="Monorepo">
+    The [Catalyst Monorepo documentation](https://www.catalyst.dev/docs/monorepo) is available as an option if you would like to manually walk through the steps outlined by the CLI. 
+    {/* Add a note as to why we would recommend the monorepo approach over the CLI */}
+</Tab>
+</Tabs>
+  </Step>
+  <Step title="Pull down the Makeswift Integration branch">
+    The Makeswift Catalyst integration is simply a branch available in the upstream GitHub repository that can be checked out locally.
+
+```
+git fetch upstream integrations/makeswift
+git checkout -b integrations/makeswift upstream/integrations/makeswift
+```
+
+</Step>
+  <Step title="Update Environment Variables">
+    You'll need to update your Catalyst application's `.env.local` file with your 
+    `Site API Key` located under your Host settings in Makeswift.
+    <Frame caption="In your Makeswift Host settings">
+    ![Makeswift Site API key](images/Site-API-key.png)
+
+</Frame>
+Add the key to your `.env.local` file with the follow key name:
+```
+MAKESWIFT_SITE_API_KEY= <YOUR_API_KEY>
+```
+<Frame caption="In your codebase">
+    ![Makeswift API env variable](images/env-local-api-key.png)
+</Frame>
+Save your `.env.local` file
+  </Step>
+<Step title="Run your development server">
+1. Install dependencies to match the current branch
+
+Ensure your local environment has the correct and updated dependencies for the specific branch you're working on by running the following command:
+
+```
+pnpm i
+```
+
+This will install and synchronize the necessary dependencies based on the package.json and pnpm-lock.yaml files for the current branch.
+Ensure that your local environment has the correct and updated dependencies required for the specific branch you are working on by installing pnpm.
+
+2. Start the development server
+
+Once the dependencies are installed, you can start the development server to run the project locally. Use the following command:
+
+```
+pnpm dev
+```
+
+</Step>
+<Step title="Update Host">
+Within the Makeswift Site Settings, update your host to whatever port your local development is running on. Typically it should run on `http://localhost:3000`
+<Frame caption="In your site settings">
+    ![Makeswift host URL](images/host-url.gif)
+</Frame>
+</Step>
+<Step title="Create a New Page">
+Create a new page within Makeswift. We recommend starting with a Blank Page.
+<Frame caption="In the builder">
+    ![Makeswift host URL](images/create-page.gif)
+</Frame>
+</Step>
+</Steps>
+
+{/* linkn to product docs quickstart for next steps. Menmtioning building components in Makeswift first before doing custom componenet dev */}
```

Comment:
Can we add this link to quickstart for next steps?

Reply:
realizing my last commit that includes this still hasn't gone through for some reason.